web_tree_dynamic_colored_field/static/description/index.html

```diff
@@ -529,13 +507,15 @@ <h1>Contributors</h1>
 <li>Guewen Baconnier &lt;<a class="reference external" href="mailto:guewen.baconnier&#64;camptocamp.com">guewen.baconnier&#64;camptocamp.com</a>&gt;</li>
 <li>Phuc Tran Thanh &lt;<a class="reference external" href="mailto:phuc&#64;trobz.com">phuc&#64;trobz.com</a>&gt;</li>
 <li>Sylvain LE GAL &lt;<a class="reference external" href="https://twitter.com/legalsylvain">https://twitter.com/legalsylvain</a>&gt;</li>
+<li>Jurgis Pralgauskis &lt;<a class="reference external" href="mailto:jurgis.pralgauskis&#64;gmail.com">jurgis.pralgauskis&#64;gmail.com</a>&gt;</li>
```

Comment:
Usually we provide our @versada.eu email

Reply:
Don't modify `index.html`. It is generated automatically.

Reply:
yes, it is taken from  `CREDITS.md`, but I got Maciejs idea :)